cypress/e2e/GUI/orderingPagination.cy.js

```diff
@@ -0,0 +1,187 @@
+        cy.get(inputName).should('be.visible').and('have.attr', 'disabled')
+
+    });
+
+
+
+    it('Verificar se é possivel ordenar por número de funcionários em ordem ascendente', () => {
+
+        const listNumberOfEmployess = []
+
+        cy.contains('th', 'Number of employees').find('button').dblclick().click()
+
+        cy.get(tableHeadNumberOfEmployees).then(tdElements => {
+
+            const totalTDs = tdElements.length;
+            for (let i = 0; i < totalTDs; i++) {
+                cy.wrap(tdElements.eq(i)).invoke('text').then(text => {
+                    listNumberOfEmployess.push(parseInt(text))
+                });
+            }
+
+        }).then(() => {
+
+            const isAscending = listNumberOfEmployess.every((value, index, arr) => index === 0 || value >= arr[index - 1]);
+            expect(isAscending).to.be.true;
+
+        });
+
+    });
+
+
+
+    it('Verificar se e possivel ordenar por número de funcionários em ordem descendente', () => {
+
+        const listNumberOfEmployess = []
+        cy.contains('th', 'Number of employees').find('button').dblclick()
+
+        cy.get(tableHeadNumberOfEmployees).then(tdElements => {
+            const totalTDs = tdElements.length;
+            for (let i = 0; i < totalTDs; i++) {
+                cy.wrap(tdElements.eq(i)).invoke('text').then(text => {
+                    listNumberOfEmployess.push(parseInt(text))
+                });
+            }
+
+        }).then(() => {
+            const isDescending = listNumberOfEmployess.every((value, index, arr) => index === 0 || value <= arr[index - 1]);
+            expect(isDescending).to.be.true;
+        });
+
+    });
+
+    it('Verificar se e possivel ordernar por tamanho em ordem ascendente', () => {
+
+        cy.get(tableHeadSize).should('be.visible').dblclick().click()
+        cy.get(firstTdTable).should('contain', 'Very Large Enterprise');
+
+    })
+
+    it('Verificar se e possivel ordernar por tamanho em ordem descedente', () => {
+
+        cy.get(tableHeadSize).should('be.visible').click()
+        cy.get(firstTdTable).should('contain', 'Small');
+
+    })
+
+
+    it('Verificar se o botão de paginação Prev esta desativado quando na primeira página', () => {
+
+        cy.get('span').invoke('text').then(text => {
+
+            const pageNumber = parseInt(text.match(/\d+/))
+
+            if (pageNumber === 1) {
+                cy.contains('button', 'Prev').should('have.attr', 'disabled')
+            }
+
+        })
+
+    })
+
+
+    it('Verificar se o botão de paginação Next esta desativado quando na ultima pagina', () => {
+
+        cy.get('span').invoke('text').then(text => {
+
+            const listNumberFinalPages = text.split(' ');
+
+            for (let i = 1; i <= listNumberFinalPages[3]; i++) {
+                cy.contains('button', 'Next').click({ force: true })
+            }
+
+            cy.contains('button', 'Next').should('have.attr', 'disabled')
```

Comment:
Veja abaixo uma forma mais simples de fazer essa verificação.

```suggestion
            cy.contains('button', 'Next').should('be.disabled')
```